### DISCUSSION TITLE
Kibana doesn´t work anymore

### DISCUSSION THREAD

**Discussion by Rhoenit** [telekom-security/tpotce#878] [Q&A]
If I want to open Kibana with port 64297 it shows me "Kibana server is not ready yet".
After reboot or an update it is still the same problem. 
All Containers are up but the Kibana Container restarts some times. 

Does anyone know what to do?

**Comment by eschultze** [comment]
Try running `unlock_es.sh` from `tpotce/bin`. ES might lock some clusters if the available disk size reaches a critical amount.
